input/fsh/SD_Bundle.fsh

```diff
@@ -2,26 +2,74 @@ Profile: MCODEPatientBundle
+[CancerRelatedComorbidities]: StructureDefinition-mcode-cancer-related-comorbidities.html
+[CancerRelatedMedicationRequest]: StructureDefinition-mcode-cancer-related-medication-request.html
+[CancerRelatedProcedureParent]: StructureDefinition-mcode-cancer-related-procedure-parent.html
+[CancerRelatedRadiationProcedure]: StructureDefinition-mcode-cancer-related-radiation-procedure.html
+[CancerRelatedSurgicalProcedure]: StructureDefinition-mcode-cancer-related-surgical-procedure.html
+[CancerStageParent]: StructureDefinition-mcode-cancer-stage-parent.html
+[ComorbidConditionCode]: StructureDefinition-mcode-comorbid-condition-code.html
+[ComorbidConditionReference]: StructureDefinition-mcode-comorbid-condition-reference.html
+[ComorbiditiesParent]: StructureDefinition-mcode-comorbidities-parent.html
+[ECOGPerformanceStatus]: StructureDefinition-mcode-ecog-performance-status.html
+[EvidenceType]: StructureDefinition-mcode-evidence-type.html
+[GeneticSpecimen]: StructureDefinition-mcode-genetic-specimen.html
+[GenomicRegionStudied]: StructureDefinition-mcode-genomic-region-studied.html
+[HistologyMorphologyBehavior]: StructureDefinition-mcode-histology-morphology-behavior.html
+[KarnofskyPerformanceStatus]: StructureDefinition-mcode-karnofsky-performance-status.html
+[LocationQualifier]: StructureDefinition-mcode-location-qualifier.html
+[MCodeCapabilityStatement]: StructureDefinition-MCodeCapabilityStatement.html
+[MCODEPatientBundle]: StructureDefinition-mcode-patient-bundle.html
+[PerformanceStatusParent]: StructureDefinition-mcode-performance-status-parent.html
+[PrimaryCancerCondition]: StructureDefinition-mcode-primary-cancer-condition.html
+[RelatedPrimaryCancerCondition]: StructureDefinition-mcode-related-primary-cancer-condition.html
+[SecondaryCancerCondition]: StructureDefinition-mcode-secondary-cancer-condition.html
+[TerminationReason]: StructureDefinition-mcode-termination-reason.html
+[TNMClinicalDistantMetastasesCategory]: StructureDefinition-mcode-tnm-clinical-distant-metastases-category.html
+[TNMClinicalPrimaryTumorCategory]: StructureDefinition-mcode-tnm-clinical-primary-tumor-category.html
+[TNMClinicalRegionalNodesCategory]: StructureDefinition-mcode-tnm-clinical-regional-nodes-category.html
+[TNMClinicalStageGroup]: StructureDefinition-mcode-tnm-clinical-stage-group.html
+[TNMPathologicalDistantMetastasesCategory]: StructureDefinition-mcode-tnm-pathological-distant-metastases-category.html
+[TNMPathologicalPrimaryTumorCategory]: StructureDefinition-mcode-tnm-pathological-primary-tumor-category.html
+[TNMPathologicalRegionalNodesCategory]: StructureDefinition-mcode-tnm-pathological-regional-nodes-category.html
+[TNMPathologicalStageGroup]: StructureDefinition-mcode-tnm-pathological-stage-group.html
+[TreatmentIntent]: StructureDefinition-mcode-treatment-intent.html
+[Tumor]: StructureDefinition-mcode-tumor.html
+[TumorMarker]: StructureDefinition-mcode-tumor-marker.html
+[TumorSize]: StructureDefinition-mcode-tumor-size.html
```

Comment:
This is _all_ the profiles in mCODE rather than just the ones we use (it's generated by a script). I've left the references in for profiles we aren't using because it's a pain to manually remove them, and they are stripped out by the Markdown processor anyway.